README.md

```diff
@@ -144,6 +160,36 @@ $ chmod +x ~/.config/hawck/scripts/example.hwk
 $ chmod -x ~/.config/hawck/scripts/example.hwk
 ```
 
+> Warning: When editing example.hwk, the macro daemon will auto-reload, but because of the [security model](#security), new keys will be ignored until you rerun `hawck-add`. You can check what keycodes are intercepted by the input daemon for our example script with:
+
+```sh
+cat /var/lib/hawck-input/keys/example.csv
+```
+
+## Persisting the configuration
+
+The input daemon should autostart, which you can enable/disable with
+
+```sh
+sudo systemctl enable hawck-inputd
+```
+
+The macro daemon will not autostart (and your journal will be spammed with `hawck-inputd: SystemError: Connection refused`). To autostart it, depending on your distribution, this may work:
+
+```sh
+echo "
+[Desktop Entry]
+Name=Hawck macro daemon
+GenericName=Key-rebinding daemon
+Comment=Intercept key presses and write Lua scripts to perform actions or modify keys
+Exec=/usr/bin/hawck-macrod
```

Comment:
Might want to change this to simply `hawck-macrod`, as it may be in `/usr/local/bin/hawck-macrod`

Reply:
Also, instead of embedding the whole `.desktop` file I think you should place it in `bin/` as `bin/hawck-macrod.desktop` and then just have `cp bin/hawck-macrod.desktop "${XDG_CONFIG_HOME:-.config}"/autostart` in the README.

Reply:
But also there's a reason why Hawck doesn't auto-install an autostart file like this. While the user is familiarizing themselves with the software they may make a mistake that causes Hawck to steal all the keyboard input (like misusing a catch-all.)

So the language around this should be something in the lines of "when you've got everything set up correctly, and you understand how it works, you may want to autostart `macrod`"